index.html

```diff
@@ -1,29 +1,265 @@
+                    </div>
+                    <ul class="new-lang">
+                        <li class="last-lang"><a href="#">HTML</a></li>
+                        <li class="last-lang"><a href="#">Bootstrap</a></li>
+                        <li class="last-lang"><a href="#">Ruby</a></li>
+                    </ul>
+                   <div class="project-button">
+                    <a href="#">
+                    <img src="icons/see2.png"></a>
+                   </div>
+                </li>
+            </ul>
+     </section>
+     <section class="about">
+        <h3 class="about-me">About me</h3>
+        <div class="about-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </div>
+            <div class="resume">
+            <ul>
+                <li><a href="#">Get my resume</a></li>
+            </ul>
+            </div>
+            <div class="second-image">
+                <img src="icons/illustration2.png">
+            </div>
+            <div class="vector">
+                <img src="icons/new-vector.png">
+            </div>
+            <div class="text-img1">
+                <div class="lang-img">
+                    <img src="icons/rect1.png">
+                </div>
+                <div class="lang-text">
+                    Language
+                </div>
+            </div>
+           <div class="language">
+                <ul> 
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+            </div>
+
+
+            <div class="text-img2">
+                <div class="frame-img">
+                    <img src="icons/rect2.png">
+                </div>
+                <div class="frame-text">
+                    Frameworks
+                </div>
+            </div>
+           <div class="framework">
+                <ul> 
+                    <li>React.js</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSpec</li>
+                    <li>Capybaras</li>
+                </ul>
+            </div>
+
+            <div class="text-img3">
+                <div class="skill-img">
+                    <img src="icons/rect3.png">
+                </div>
+                <div class="skill-text">
+                    Skills
+                </div>
+            </div>
+           <div class="skills">
+                <ul> 
+                    <li>Database Management</li>
+                    <li>Version Control</li>
+                    <li>CLI</li>
+                    <li>API Design</li>
+                    <li>Web Development</li>
+                </ul>
+            </div>
```

Comment:
- [ ] Please make sure that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`. As a best practice, it's always a good idea to combine a lit of items that are related in a `ul` and `li` tags. You can still use a `ul` for each technology and its items.